__tests__/app.test.js

```diff
@@ -356,5 +357,33 @@ describe("GET /api/articles/:article_id", () => {
   });
   
 
+  describe("GET can sort by different fields", () => {
+    test("200: /api/articles?sort_by=title", () => {
+      return request(app)
+        .get("/api/articles?sort_by=title")
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+          expect(articles).toBeSortedBy("created_at", {ascending: true ,coerce: true,});
+        });
+    });
+    test("200: /api/treasures?sort_by=author", () => {
+      return request(app)
+        .get("/api/articles?sort_by=author")
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+          expect(articles).toBeSortedBy("author",{descending: true ,coerce: true,});
+        });
+    });
```

Comment:
This looks like a good test, but I am unsure it works as expected given the first test